src/BitcoinHelpers.js

```diff
@@ -344,9 +380,9 @@ const BitcoinHelpers = {
      * @param {number} requiredConfirmations A number of required
      *        confirmations below which this function will return null.
      *
-     * @return {Promise<number>} A promise to the current number of
-     *         confirmations for the given `transaction`, iff that transaction has
-     *         at least `requiredConfirmations` confirmations.
+     * @return {Promise<number?>} A promise to the current number of
+     *         confirmations for the given `transaction`, iff that transaction
```

Comment:
iff -> small typo from previous

Reply:
I believe this was used intentionally here (to mean [if and only if](https://en.wikipedia.org/wiki/If_and_only_if)).

Reply:
Nice. I've never seen that before, thanks!